cockpit-frontend/src/redux/ormSlice.jsx

```diff
@@ -12,6 +12,10 @@ export const fetchAllMvps = createAsyncThunk('mvps/fetchAllMvps', async () => {
   const allMvps = await MvpService.getAll();
   return allMvps.data;
 });
+export const getOneMvp = createAsyncThunk('mvps/getOneMvp', async (id) => {
```

Comment:
what do you think about replacing getOneMvp by fetchUpdatedMvp() so that it match the existing function fetchAllMvps?